### DISCUSSION TITLE
What is the criteria of being a maintainer of a popular open source project?

### DISCUSSION THREAD

**Discussion by shd101wyy** [community/community#19133] [Copilot Conversations]
I participated in the copilot technical  preview. I enjoyed using copilot a lot. It is quite helpful to my works. 🌝 

I have multiple open source projects with more than 500 stars each. But I am not qualified as the maintainer of popular open source projects. So I am curious about your criteria. 🌝

I also failed to subscribe to a yearly plan as it showed my account is locked. Any suggestions on that?

Thanks!

**Comment by melvincwng** [comment]
@shd101wyy, for the criteria on whether you qualify to use GitHub Copilot at no cost, please refer to [this link](https://github.com/pricing#faq-copilot).

Regarding your account being locked and being unable to purchase GitHub Copilot, please raise a ticket to [Github Support](https://support.github.com/).

Hope this clarifies.

**Reply by melvincwng** [reply]
@shd101wyy, if this reply has helped you, please kindly 'Mark as answer' so that others know this issue/question has been addressed.

Thank you 😊

**Reply by shd101wyy** [reply]
So the criteria is like a “blackbox” 🌝 That's why I am asking here.

I will contact the support. Thanks

**Reply by melvincwng** [reply]
@shd101wyy, glad to have helped :)

**Reply by guoquan** [reply]
Op is obviously not asking for unlocking account, but a transparent, auditable, non-blackbox award process.
